cloudify_rest/tasks.py

```diff
@@ -102,8 +103,11 @@ def _execute(params, template_file, instance, node, save_path=None,
         merged_params = {}
         merged_params.update(node.properties.get("params", {}))
         merged_params.update(params)
+        merged_auth = node.properties.copy()
+        if auth:
+            merged_auth.update(auth)
```

Comment:
@0lvin you are trying to update `merged_auth` which is a dict with `boolean` value `auth` , I think this should be updated like this

```
if auth:
    merged_auth.update({YOUR_KEY: auth})
```

Reply:
Boolean - is only placeholder for something that is not true/filled by default.